src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -416,13 +418,12 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
           flags: [],
           clientVersion: config.clientVersion,
           enableExperimental: mapPreviewFeatures(this._previewFeatures),
-          useUds: internal.useUds,
```

Comment:
Why removing useUds? Some users need it, that would break for them

Reply:
Tim requested that I remove UDS, I've been a bit hesitant about that so that's why it's half in half out. I think you are right, I'll make sure it remains

Reply:
Seems we could remove later to avoid breaking change indeed, doesn't cost us much to keep it for now